server/test/src/main/java/org/eclipse/lsp/cobol/test/engine/UseCasePreprocessorListener.java

```diff
@@ -166,13 +170,14 @@ private Consumer<CpyIdentifierContext> processCopybook(
           CopybookStatementContext ctx, Map<String, List<Location>> copybookUsages) {
     return it -> {
       String dialect = it.cpyDialect() == null ? "" : it.cpyDialect().getText();
-      processCopybookToken(
+      String finalName = processCopybookToken(
```

Comment:
Is `finalName` going to be equal `StringUtils.trimQuotes(it.cpyName().getText().toUpperCase())` ?